pkgs/tools/system/supercronic/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, buildGoModule, fetchFromGitHub, python3, bash, coreutils }:
+
+buildGoModule rec {
+  pname = "supercronic";
+  version = "0.2.1";
+
+  excludedPackages = [ "cronexpr/cronexpr" ];
+  checkInputs = [ python3 bash coreutils ];
+  vendorHash = "sha256-oSwow6Ajh4HWrr+hfjlJIp1DOBCvN1Khm4tseWokxoc=";
+
+  src = fetchFromGitHub {
+    owner = "aptible";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-vaRoWXrCi7dLofluldiCT1yH0yCa4j1g+llv1MZDhOY=";
+  };
+
+  postConfigure = ''
+    # There are tests that set the shell to various paths
+    substituteInPlace cron/cron_test.go --replace /bin/sh ${bash}/bin/sh
+    substituteInPlace cron/cron_test.go --replace /bin/false ${coreutils}/bin/false
+  '';
+
+  preCheck = ''
+    # There's a test that tries to run python
+    export PATH=${python3}/bin:$PATH
```

Comment:
Why not add add `python3` to `checkInputs`?

Reply:
> It's already present in `checkInputs`, but I still had to explicitly add it to the `PATH` to get the test to pass. Let me know if there's something I'm missing here.

I think this will work if you'd use `nativeCheckInputs` instead. A while ago, `mkDerivation` and other builders started supporting only `nativeCheckInputs` instead of `checkInputs`.